populist-web.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {}
+}
```

Comment:
keep this local please

Reply:
I'll add to git ignore

Reply:
File removed / *.code-workspace added to gitignore